aptos-move/aptos-vm/src/verifier/resource_groups.rs

```diff
@@ -102,8 +123,17 @@ pub(crate) fn validate_module_and_extract_new_entries(
             (BTreeMap::new(), BTreeMap::new())
         };
 
+    // INVARIANT:
+    //   No need to charge gas for this module access: this is an old version of the module and it
+    //   was charged for when processing the publish request (if it exsited).
```

Comment:
typo: *existed

Reply:
what does the comment mean: this is an old version of the module? the variable is called "new_module", also helps to know how does traversal context distinguish old and new. Is it the name?

Reply:
We use `new_module.name()` to get ID of the module and then fetch the old version corresponding to this name, so this is the old one. Changed the comment to make this clearer